### PR TITLE
Changelog main issue readme

### PR DESCRIPTION
**Pull request by RadValentin**
Cleaned up `README.md` and added new info according to issue #1

### CODE REVIEW

**Comment by RadValentin** [comment]
Please review the changes and let me know if anything else is needed
cc: @catalinmiron @skidding

**Comment by ovidiuch** [comment]
> A widget that provides you with an easy way to send transparent status updates to your customers by tracking a branch on your GitHub repository. Closed issues show up as notifications inside the widget.

Not really content with this but we can change it at any time in the future. Try to express as much as possible with as few words as possible. E.g. "Use GitHub issues to communicate app updates directly to your customers."

> changelog wireframe
- Link - This doesn't express anything. What does the label express? "New updates!" maybe. It's a button that _screams_
- Other - Why promote misc labels when we can enforce direct use-cases. I'd use "Feature" and "Bug", or even better "Fixed", "Improved", etc.
- Changelog (23) - changelog is a codename. I'm thinking "Updates ready (23)"

File: README.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
+
+![changelog wireframe](https://dl.dropboxusercontent.com/u/42934143/images/changelog.png)
+
 ### Notification icon ###
 This is what shows up by default, before the user has had a chance to interact with the widget.
 
 **Behaviour**
-- when no unread notifications are present its styling should reflect this, possibly grayed out or with some default coloring.
-- when new notifications are received the icon styling changes to a different, brighter color. Possibly with a glow effect. In addition, a counter will show up displaying the number of new notifications.
+- when no unread notifications are present the icon won't be shown.
```
Comment (ovidiuch) [inline README.md]: This sentence feels weird, shouldn't it be reversed? "The icon won't be shown until updates are ready."

PS. Try to start all sentences with a capital letter for consistency.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
+- when new notifications are received the icon becomes visible. In addition, a counter will show up displaying the number of new notifications.
```
Comment (ovidiuch) [inline README.md]: If you nail the previous point right, you don't need to repeat yourself about the icon being visible only when update are ready. Instead, this point could only emphasize on one thing. "In addition to the visible icon, a small counter will also show up when updates arrive, displaying their count."

**Comment by ovidiuch** [comment]
Finished review for now. Try to revise the doc with consideration to my comments and merge it. We'll have plenty of time to tweak it once we have a working prototype.

**Comment by RadValentin** [comment]
Updated the readme according to your feedback, let me know what you think.

cc: @skidding

**Pull request merged**